python/lsst/ts/criopy/m1m3/force_calculator.py

```diff
@@ -324,10 +353,16 @@ def __add__(self, obj2: Any) -> Self:
             return NotImplemented
 
     def __init__(self, config_dir: None | str | pathlib.Path = None):
+        self._fam_computation: list[pd.DataFrame] = []
+        self._acceleration_computation: list[pd.DataFrame] = []
+        self._velocity_computation: list[pd.DataFrame] = []
```

Comment:
Why do you need these three lines here? They look unrelated to this PR

Reply:
It was reverted, as it was really not needed.